git-subhistory.sh

```diff
@@ -16,10 +16,11 @@ test $# = 0 && set -- -h
 
 OPTS_SPEC="\
 git-subhistory split <subproj-path> (-b | -B) <subproj-branch>
-git-subhistory merge <subproj-path> <subproj-branch>
+git-subhistory merge <subproj-path> (-a) <subproj-branch>
 --
 q,quiet         be quiet
 v,verbose       be verbose
+a,auto          use auto-generated commit message without prompting user
```

Comment:
That's a good thought, but for now I think it's best to handcode this one option.  Some of the other options may or may not make sense, and I haven't thought through all the implications of supporting them in general.